docs/concepts/configuration/assign-pod-node.md

```diff
@@ -206,6 +206,31 @@ If defined but empty, it means "all namespaces."
 All `matchExpressions` associated with `requiredDuringSchedulingIgnoredDuringExecution` affinity and anti-affinity
 must be satisfied for the pod to schedule onto a node.
 
+#### More Practical Usage
+
+Interpod Affinity and AnitAffinity can be even more useful when they are used with higher
+level collections such as ReplicaSets, Statefulset, Deployments, etc.  One can easily configure that a set of workloads should
+be co-located in the same defined topology, eg., the same node. 
+
+##### Always co-located in the same node
+
+A web application has an in-memory cache such as redis, one may prefer to co-locate their web server along with the cache as much as possible so that they have minimum latency like below.
+
+|       node-1         |       node-2        |       node-3       |
+|:--------------------:|:-------------------:|:------------------:|
+| *webserver-1*        | *webserver-2*       | *webserver-3*      |
+|  *cache-1*           |   *cache-2*         |   *cache-3*        |
+
```

Comment:
Sure. StatefulSets or Deployments are totally fine. The affinity/anti-affinity rules are the same for all of them and that's the important piece of the config in this example. I have no particular preference about type of the collection.